WcaOnRails/app/models/website_contact.rb

```diff
@@ -2,4 +2,16 @@
 
 class WebsiteContact < ContactForm
   attribute :message, validate: true
+
+  attr_accessor :inquiry
+  attr_accessor :competition_id
+
+  # Override the `to_mail` validation, to show errors for `inquiry` instead.
```

Comment:
Well, it may fail in two cases:
- there is no `to_email` at all (this is the case if someone doesn't specify `inquiry` in this case)
- we have a typo in our code that sets it to an invalid email

We could certainly have presence validation on `inquiry` instead and drop the validation on `to_email` (for all contact forms), assuming that we ensure it's validity on our own. The `to_email` was here, that's why I left it ^^ Thoughts?

Reply:
Ah, gotcha. I think part of my confusion is coming from the fact that the mapping between inquiry to destination email address is in the controller, instead of over here. Have you considered maybe moving that mapping into this `WebsiteContact` model? I think that could make it clearer what's going on. I think I'd implement this by either implementing our own `def to_email` method, or if that proves tricky perhaps by explicitly setting `to_email` in a before validation block.

Reply:
I thought about `before_validation` callback, but `WebsiteContact` is not an `ActiveRecord` (just includes some validations), so it doesn't support all those neat hooks. Overriding `to_email` sounds like a plan though, gonna give it a shot!